android/app/src/main/java/app/organicmaps/bookmarks/BookmarkCategoriesAdapter.java

```diff
@@ -26,6 +26,7 @@ public class BookmarkCategoriesAdapter extends BaseBookmarkCategoryAdapter<Recyc
   private final static int TYPE_CATEGORY_ITEM = 1;
   private final static int TYPE_ACTION_ADD = 2;
   private final static int TYPE_ACTION_IMPORT = 3;
+  private final static int TYPE_ACTION_EXPORT = 4;
```

Comment:
EXPORT_ALL_AS_KMZ?

Users ask to export files in GPX too, although it's more about exporting only one file, we may also add exporting of all files as GPX in GPZ archive.

Or we'll add a format selection as a submenu after pressing the export button?

Reply:
Fixed